### PR TITLE
OMID-234 Bump SnakeYaml version to 1.33

### PR DESCRIPTION
**Pull request by richardantal**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1071,6 +1071,13 @@
                 <version>${netty4.version}</version>
             </dependency>
             
+            <!-- set version for snakeyaml transitive dependencies -->
```
Comment (stoty) [inline pom.xml:1074]: If we add a dependencyManagement entry, then we ought to remove the version from the dependency.

File: common/src/main/java/org/apache/omid/YAMLUtils.java [hunk 2]
```diff
@@ -17,9 +17,9 @@
  */
 package org.apache.omid;
 
+import org.apache.commons.beanutils.BeanUtils;
```
Comment (stoty) [inline common/src/main/java/org/apache/omid/YAMLUtils.java:20]: No need to reformat the code for a version bump.

**Review by stoty — CHANGES_REQUESTED** [review]

**Review by stoty — APPROVED** [review]
+1 LGTM

**Comment by stoty** [comment]
I can see a test error, but that was already present earlier.

**Comment by richardantal** [comment]
Thanks @stoty for the review!

**Pull request merged**